test_project/lib/extends_with_mixin/getter.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:autoequal/autoequal.dart';
+import 'package:equatable/equatable.dart';
+
+part 'getter.g.dart';
+
+@autoequalMixin
+class Getter extends Equatable with _$GetterAutoequalMixin {
+  const Getter();
+
+  set one(String newValue) => print(newValue);
```

Comment:
Same here. It seems that it should be getter instead of a setter

```suggestion
  String get one => 'one';
```